### DISCUSSION TITLE
Pair to MacOS setup without Visual Studio for Mac

### DISCUSSION THREAD

**Discussion by ward-horsfall** [unoplatform/uno#17095] [Q&A]
Hi

Given Microsoft is dropping Visual Studio for MacOS.

What are the specific steps I need to do instead?

Thanks 
Ward

**Comment by carldebilly** [comment]
It depends which platform you want to target...

First, you can check the _Getting Started_ documentation: <https://platform.uno/docs/articles/get-started.html>

From that page, you can infer those:
* **iOS**: If you want to target iOS, you can build on Windows using any IDE (Visual Studio, VS Code or Rider) or, on a Mac, using Rider or VS Code.
* **Mac Catalyst** or **Skia Desktop** (macOS): Using VS Code (both on Windows or Mac) and, eventually, Rider
